### ISSUE TITLE
Intellisense not working in second window

### ISSUE THREAD

**Issue by AutomationMusician** [OmniSharp/omnisharp-vscode#3261]
I use the C# extension regularly in order to run code, view where methods are referenced, etc., but when I open a second window for Visual Studio Code, those features do not work in the new window (as seen in the image below). This feature would be very helpful for looking through multiple C# files at once. Is this a common problem or only something that only I am running into? Is there a fix that I can do?

![image](https://user-images.githubusercontent.com/47127655/64343765-20f0d980-cfbb-11e9-8402-e98f11d3fee5.png)

Thank you for your help!

Versions and such:
Windows 10 Enterprise 64-bit
Visual Studio Code: version 11.37.1
C# Extension: version 1.21.0

**Comment by akshita31** [comment]
@AutomationMusician I use the extension regularly and definetly that is not the case. In the window on the right have you opened `Program.cs` along with the associated csproj or just the file ?

If the vscode workspace that is opened doesn't include a project, the files in the workspace are treated as "Miscellaneous files". In such a case, intellisense should light up once you start editing the files. Ca you try to edit the file and see if intellisense works ?

**Comment by AutomationMusician** [comment]
I was not in a workspace containing a .csproj file, so I tried editing the file and unfortunately intellisense did not start working. However, when I opened a folder containing a .csproj file and waited a few moments, intellisense started working. All of the files that I am working on are associated with a project, so this will not be a problem.
Thanks for your response. It was very helpful!
